plenum/test/common/test_replicas_suspicious.py

```diff
@@ -0,0 +1,34 @@
+from plenum.common.constants import DOMAIN_LEDGER_ID
+
+from plenum.test.helper import waitForViewChange
+from stp_core.loop.eventually import eventually
+
+
+def test_nodes_make_view_change_only_on_master_suspicious(
+        looper, txnPoolNodeSet):
+    old_view = txnPoolNodeSet[0].viewNo
+
+    primary1 = txnPoolNodeSet[0].replicas[0]
+    primary2 = txnPoolNodeSet[1].replicas[1]
+    assert primary1.isPrimary is True
+    assert primary2.isPrimary is True
+
+    primary1.batchDigest = lambda reqs: 'asd'
+    primary2.batchDigest = lambda reqs: 'asd'
+
+    non_primary1 = txnPoolNodeSet[0].replicas[1]
```

Comment:
Better variable name will be `non_primary_backup`

Reply:
ok